starter-code/index.html

```diff
@@ -1,21 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
+<link rel="stylesheet" type="text/css" href="../starter-code/css/main.css">
 <head>
   <title>Instagram</title>
 </head>
 <body>
+ <div class="upper-block">
   Instagram
+  <div class= "phone-picture">
+  <figure>
+   <img src="../starter-code/images/phones.png">
+  </figure>
+</div>
+ <div class= "logo">
+  <figure>
+   <img src="../starter-code/images/brand.png">
+  </figure>
+ </div>
+ <div class='home-button'>
+   <img src="../starter-code/images/home.png">
+   Log in
 
-  Capture and Share the World's Moments
+ </div>
+</div>
+<div class="frame">
+  <div class= "text-in-frame">
+    <div class="frame-header">
+     Capture and Share the World's Moments
+   </div>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+  Instagram is a <b>fast</b>, <b>beautiful</b> and <b>fun</b> way to share your life with friends and family.<br>
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+
+
+  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.<br>
 
   Oh yeah, did we mention it's free?
+  </div>
+  <div class ="ios-android">
+    <img src="..\starter-code\images\badge-android.png">
+    <img src="..\starter-code\images\badge-android.png">
+  </div>
+</div>
+
+<div class="footer">
```

Comment:
You should turn this element into an unordered list with multiple list items to give them all proper spacing.